source/memory/memory.go

```diff
@@ -30,6 +30,23 @@ type Source struct{}
 // Name returns an identifier string for this feature source.
 func (s Source) Name() string { return "memory" }
 
+func GetChoice(path string) (string, error) {
+	val, err := ioutil.ReadFile(path)
+	if err != nil {
+		return "", fmt.Errorf("failed to read '%s': %v", path, err)
+	}
+
+	// Split up the choices
+	allChoices := strings.Fields(string(val))
+
+	for _, choice := range allChoices {
```

Comment:
nit: you don't necessarily need allChoises. Just
for _, choice := range strings.Fields(string(val))
??

Reply:
Done. Thanks.